test/BlockManager.js

```diff
@@ -1144,99 +1144,43 @@ describe('BlockManager', function () {
       staker = await stakeManager.getStaker(stakerIdAcc14);
-      influence = (await stakeManager.getInfluence(influencerIds[0]));
-      iteration = await getIteration(voteManager, stakeManager, staker, influence);
-      proposedBlocksInfluence.push(influence);
-      proposedBlocksIteration.push(iteration);
-      await blockManager.connect(signers[14]).propose(epoch,
-        [100, 201, 300, 400, 500, 600, 700, 800, 900],
-        iteration,
-        influencerIds[0]);
-
-      const stakerIdAcc15 = await stakeManager.stakerIds(signers[15].address);
-      staker = await stakeManager.getStaker(stakerIdAcc15);
-      influence = (await stakeManager.getInfluence(influencerIds[2]));
-      iteration = await getIteration(voteManager, stakeManager, staker, influence);
-      proposedBlocksInfluence.push(influence);
-      proposedBlocksIteration.push(iteration);
-      await blockManager.connect(signers[15]).propose(epoch,
-        [100, 201, 300, 400, 500, 600, 700, 800, 900],
-        iteration,
-        influencerIds[2]);
-
-      const stakerIdAcc16 = await stakeManager.stakerIds(signers[16].address);
-      staker = await stakeManager.getStaker(stakerIdAcc16);
-      influence = (await stakeManager.getInfluence(influencerIds[1]));
-      iteration = await getIteration(voteManager, stakeManager, staker, influence);
-      proposedBlocksInfluence.push(influence);
-      proposedBlocksIteration.push(iteration);
-      await blockManager.connect(signers[16]).propose(epoch,
+      iteration = await getIteration(voteManager, stakeManager, staker, influenceSmallest);
+      const tx = blockManager.connect(signers[14]).propose(epoch,
         [100, 201, 300, 400, 500, 600, 700, 800, 900],
         iteration,
         influencerIds[1]);
-
-      const proposedBlocksLength = await blockManager.getNumProposedBlocks(epoch);
-
-      const sortedBlockIds = [];
-      let pushAt;
-      for (let i = 0; i < proposedBlocksIteration.length; i++) {
-        if (sortedBlockIds.length === 0) {
-          sortedBlockIds.push(0);
-          // eslint-disable-next-line no-continue
-          continue;
-        }
-        pushAt = sortedBlockIds.length;
-        for (let j = 0; j < sortedBlockIds.length; j++) {
-          if (proposedBlocksInfluence[sortedBlockIds[j]] < proposedBlocksInfluence[i]) {
-            pushAt = j;
-            break;
-          }
-          if (proposedBlocksIteration[sortedBlockIds[j]] > proposedBlocksIteration[i]) {
-            pushAt = j;
-            break;
-          }
-        }
-        sortedBlockIds.push(i);
-        for (let j = sortedBlockIds.length - 1; j > (pushAt); j--) {
-          sortedBlockIds[j] = sortedBlockIds[j - 1];
-        }
-        sortedBlockIds[pushAt] = i;
-      }
-
-      for (let i = 0; i < proposedBlocksLength; i++) { // 20341
-        sortedProposedBlockId = await blockManager.sortedProposedBlockIds(epoch, i);
-        assertBNEqual(sortedBlockIds[i], sortedProposedBlockId, 'Not sorted properly');
```

Comment:
???

Reply:
we have it checked it one test written just above